TransformationSystem/Agent/TransformationCleaningAgent.py

```diff
@@ -356,6 +358,9 @@ def cleanTransformationLogFiles(self, directory):
     self.log.verbose("Removing log files found in the directory %s" % directory)
     res = returnSingleResult(StorageElement(self.logSE).removeDirectory(directory, recursive=True))
     if not res['OK']:
+      if cmpError(res['Message'], errno.ENOENT):  # No such file or directory
```

Comment:
@chaen I think this is what you meant.

Reply:
```cmpError(res, errnoENOENT)``` is even better because it will check first the error number if it exists

Reply:
Done.